src/stylesheets/utility/_body.scss

```diff
@@ -0,0 +1,7 @@
+body {
+  font-family: 'Geomanist Regular';
+  font-size: rem(18);
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Done

Reply:
fwiw, was discussing this with a designer and not sure if I agree or disagree. it's a little troubling because that 7-year-old post is pretty much the lone voice that i could find taking this stance. on the other hand, it's a good default to not broadly apply something like this. i only raise all this bc there will inevitably be times where, with certain fonts, the designer requests applying this broadly and maybe that's okay 🤷‍♂️